docs/src/app/components/pages/components/dialog.jsx

```diff
@@ -72,6 +76,12 @@ export default class DialogPage extends React.Component {
             desc: 'Set to true to have the dialog automatically open on mount.',
```

Comment:
@oliviertassinari until we do the next minor release (v0.15.x), we shouldn't introduce breaking changes. So if we want to merge this to `master` now, we should introduce some kind of a deprecation warning.

As far as I'm aware, we haven't used deprecation warnings before. So @JorgenEvens you're free to set the convention here I suppose :-)